stumpy/core.py

```diff
@@ -1843,6 +1888,13 @@ def preprocess(T, m, copy=True, M_T=None, Σ_T=None, T_subseq_isconstant=None):
         A boolean array that indicates whether a subsequence in `T`
         is constant (True)
 
+    isconstant_custom_func : object, default None
+        A custom, user-defined function that returns boolean numpy ndarray that indicate
+        if a subsequence is constant or not. It takes two arguments, `a`, a 1-D array,
+        and `w`, the window size, and may have default arguments if needed. When `None`,
+        this will be default to the function `core._rolling_isconstant`. This parameter
+        does not recompute `T_subseq_isconstant`.
+
```

Comment:
Is this good?

Reply:
What do you mean "This parameter does not recompute `T_subseq_isconstant`"? If it's a function that needs to be called, it should compute `T_subseq_isconstant`, right?